### ISSUE TITLE
How you got the Stanford's library to work?

### ISSUE THREAD

**Issue by Mr-Welcome1** [tgreen443/niuniu_assignment#2]
Sorry for creating an issue but I didn't know how to contact you, 

Would you my mind telling me how you got the stanford library working? What IDE you are using? Which version of the course you are going with?
I'm currently planning to take [this](https://see.stanford.edu/Course/CS106B) version of the course, But kinda it seems very outdated and it's hard to follow up.

Appreciate any recommendation or help, Thanks...

**Comment by tgreen443** [comment]
you can download all the resources on their newest webset [https://web.stanford.edu/class/archive/cs/cs106b/cs106b.1228/],and) the resources have shown you how to use IDE(QT),and how to compile the library(I run it on Win11 and it's easier than that on Linux )。Apologize that I cannot give you too much advice because I just started to learn to code. All your best!

**Comment by tgreen443** [comment]
> Sorry for creating an issue but I didn't know how to contact you,
> 
> Would you my mind telling me how you got the stanford library working? What IDE you are using? Which version of the course you are going with? I'm currently planning to take [this](https://see.stanford.edu/Course/CS106B) version of the course, But kinda it seems very outdated and it's hard to follow up.
> 
> Appreciate any recommendation or help, Thanks...

you can download all the resources on their newest webset [https://web.stanford.edu/class/archive/cs/cs106b/cs106b.1228/],and) the resources have shown you how to use IDE(QT),and how to compile the library(I run it on Win11 and it's easier than that on Linux )。Apologize that I cannot give you too much advice because I just started to learn to code. All your best

**Comment by Mr-Welcome1** [comment]
I tried to get it to work on Linux, (no success)
Then I moved to Windows, as you mentioned, worked just fine. You were really a helper, many thanks.😍

**Comment by tgreen443** [comment]
> I tried to get it to work on Linux, (no success) Then I moved to Windows, as you mentioned, worked just fine. You were really a helper, many thanks.😍

yes,when you run it on linux,you need to consider switching the gcc version,the linux version ,the  third party and many other things. It's easier to get start on windows and focus on the content of the course rather than building the environment . Wish you good luck!
